src/shapes/FocusedInputShape.js

```diff
@@ -5,4 +5,10 @@ import {
   END_DATE,
 } from '../../constants';
 
-export default PropTypes.oneOf([START_DATE, END_DATE]);
+export default function(props, propName, componentName){
+  const item = props[propName];
+  if (item === START_DATE || item === END_DATE || item === ''){
+    return null;
+  }
+  return new Error(`${propName} must be an empty string, "${START_DATE}", or "${END_DATE}"`);
```

Comment:
Wait, but currently we accept `START_DATE`, `END_DATE`, and `null`... do we need something else? Why do you want something else?